bundles/org.eclipse.core.resources/src/org/eclipse/core/internal/watson/ElementTreeWriter.java

```diff
@@ -79,11 +79,17 @@ public Object readData(IPath path, DataInput input) {
 	}
 
 	/**
-	 * Sorts the given array of trees and The sort order is written to the given
-	 * output stream.
+	 * Writes indexes of sorted trees to output stream.
+	 *
+	 * @return sorted array of trees.
+	 * @throws IOException if sorting can not be done.
+	 * @see SaveManager#sortTrees
 	 */
-	protected ElementTree[] sortTrees(ElementTree[] trees, DataOutput output) throws IOException {
+	protected ElementTree[] writeSortedTrees(ElementTree[] trees, DataOutput output) throws IOException {
```

Comment:
Although unlikely since this method is internal you might consider to leave the method name. Since it is protected there *might* be unknown subclasses. Just think about if it is worth renaming. The documentation is already clear about what happens.

Reply:
I wouldn't care, this is so low level, I don't expect anyone is messing with that code.

Reply:
Fine with me.